tsfile/src/main/java/org/apache/iotdb/tsfile/read/expression/ExpressionType.java

```diff
@@ -19,5 +19,5 @@
 package org.apache.iotdb.tsfile.read.expression;
 
 public enum ExpressionType {
-  AND, OR, SERIES, GLOBAL_TIME
+  AND, OR, SERIES, GLOBAL_TIME, TRUE
```

Comment:
pls add comment why a TRUE is added here, to let new committers know that.

Reply:
fixed